1.2.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+
+// Task: ����������� ������ ����������>
+
+size_t n = 0;
+long* arr = NULL;
+
+void sortType()
+{
+	size_t i;
+	printf("Array size: ");
+	scanf_s("%u", &n);
+
+	arr = (long*)calloc(n, sizeof(long));
+
+	for (i = 0; i < n; i++) 
+	{
+		printf("Array[%d] =  ", i);

```

Comment:
```suggestion
		printf("Enter the number #%d: ", i);
```
С пользователем нужно на человеческом разговаривать.
(-1 балл)